Makefile

```diff
@@ -402,6 +402,7 @@ prepare_collection_venv:
 COLLECTION_TEST_DIRS ?= awx_collection/test/awx
 COLLECTION_PACKAGE ?= awx
 COLLECTION_NAMESPACE ?= awx
+COLLECTION_INSTALL = ~/.ansible/collections/ansible_collections/$(COLLECTION_NAMESPACE)/awx
```

Comment:
i think this should be `~/.ansible/collections/ansible_collections/$(COLLECTION_NAMESPACE)/$(COLLECTION_PACKAGE)`

Reply:
There are a couple of places that should have been done but wasn't. I'll make an attempt, but I don't expect that I will get them all.